src/uwtools/cli.py

```diff
@@ -704,7 +722,7 @@ def _add_arg_verbose(group: Group) -> None:
 # Support
 
 
-def _abort(msg: str) -> None:
+def _abort(msg: str) -> NoReturn:
```

Comment:
The `NoReturn` type signals to typecheckers that this function will not return at all. It prevents e.g. "may not be defined" warnings about variables accessed after the call to this function: The typechecker knows that it doesn't matter that they may not be defined, because they will never be references, because the call to this function will never return.